### PR TITLE
Split logging into 2 packages

### PR DESCRIPTION
**Pull request by JakeGinnivan**
The second package is when we need to write to multiple streams which is not the recommended pino way.

This second package will hopefully be deprecated down the track when we no longer need to log to file

### CODE REVIEW

**Review by andyjdavis — APPROVED** [review]

File: packages/logging-multi/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+## Usage
+
+#### Logger
+
+```ts
+import logging from 'etrigan/logging-multi'
+
+// If a logfile is specified pino-multi-stream is configured
+// otherwise it will use plain pino for speed reasons
+const log = logging.createLogger('MyService', {
```
Comment (cyruseftos) [inline packages/logging-multi/README.md]: The createLogger function doesn't have a name param. It looks like all config is in the options object 👍 
```suggestion
const log = logging.createLogger({name: 'MyService, logfile: process.env.LOG_FILE})
```

File: packages/logging-multi/src/create-logger.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        level,
+        enabled: opts.enabled !== false,
+        serializers: {
+            req: asReqValue,
+            res: asResValue,
+            err: pino.stdSerializers.err,
+        },
+    }
+
+    /* eslint-disable @typescript-eslint/no-var-requires */
```
Comment (antonjb) [inline packages/logging-multi/src/create-logger.ts]: Not sure this is used?

File: packages/logging-multi/README.md (hunk 1 above)
Comment (JakeGinnivan) [reply packages/logging-multi/README.md]: Sorted

File: packages/logging-multi/src/create-logger.ts (hunk 2 above)
Comment (JakeGinnivan) [reply packages/logging-multi/src/create-logger.ts]: Done

**Review by cyruseftos — APPROVED** [review]

**Pull request merged**